app/controllers/admin/enrollments_controller.rb

```diff
@@ -6,7 +6,7 @@ def index
 
   def send_invitation_email
     UserMailer.invite(enrollment).deliver_later
-    enrollment.update_attribute :has_sent_invitation_email, true
+    enrollment.update(has_sent_invitation_email: true, invitation_sent_time: Time.now)
```

Comment:
we don't need `has_sent_invitation_email` anymore. just test `invitation_sent_time.nil?`